tests/src/python/test_qgsmapboxglconverter.py

```diff
@@ -860,7 +860,7 @@ def testScaledLabelShieldIcon(self):
         renderer, has_renderer, labeling, has_labeling = QgsMapBoxGlStyleConverter.parseSymbolLayer(style, context)
         self.assertTrue(has_renderer)
         size = renderer.symbol().symbolLayers()[0].size()
-        self.assertEqual(size, 0.5)
+        self.assertEqual(size, 1.0)
         dd_properties = renderer.symbol().symbolLayers()[0].dataDefinedProperties()
```

Comment:
I'd expect a few more test changes then just this? Is the actual scenario covered by this fix included in tests?

Reply:
I still have to add extra tests